keyboards/trainpad/info.json

```
@@ -0,0 +1,28 @@
+{
+    "keyboard_name": "TrainPad",
+	"manufacturer": "Ananya Kirti",
+	"url": "",
+	"maintainer": "Ananya Kirti",
```

This is meant to be the maintainer's GitHub username.

```suggestion
	"maintainer": "AnanyaKirti",
```